fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1226,7 +1227,7 @@ void continuationDuringRepartitioningTest(boolean isGrouped,
                         .setDocId(1000L + i)
                         .setIsSeen(true)
                         .setText("A word about what I want to say")
-                        .setTimestamp(start + i * 100)
+                        .setTimestamp(uniqueTimestamps ? start + i * 100 : start)
```

Comment:
We should also test with a few (e.g. 3) distinct timestamps.
And we should make the timestamps have a different ordering from primary key.
Perhaps also not having the ordering of timestamps align with the insert order.

This might start to warrant adding some randomized arguments to this test, if that's easy enough, so that we don't run all permutations every time. But I don't think that's worth it right now if it means spending a bunch of time trying to get the validation to handle all the possibilities.

Reply:
Mixing the order will require a bit more sophisticated tracking of expectations. I'll probably defer this to a following update.